test/RatesMapping.sol

```diff
@@ -1,9 +1,23 @@
-// SPDX-License-Identifier: MIT
-pragma solidity 0.8.19;
+// SPDX-FileCopyrightText: © 2025 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-contract RatesMapping {
+pragma solidity 0.8.24;
```

Comment:
Let's not use fixed versions in the files themselves, but rather fix it in `foundry.toml`.
It makes it easier to import.
```suggestion
pragma solidity ^0.8.24;
```